pkg/signature/ed25519ph.go

```diff
@@ -46,6 +46,11 @@ func LoadED25519phSigner(priv ed25519.PrivateKey) (*ED25519phSigner, error) {
 	}, nil
 }
 
+// ConvertToED25519 creates a ED25519SignerVerifier from a ED25519phSignerVerifier
+func (e ED25519phSignerVerifier) ConvertToED25519() (*ED25519SignerVerifier, error) {
```

Comment:
nit: I'd call this `ToED25519SignerVerifier` to be specific with which interface it returns